nextcloudappstore/api/v1/release/info.xsd

```diff
@@ -369,6 +369,15 @@
 
     <xs:simpleType name="licence">
         <xs:restriction base="xs:string">
+            <xs:enumeration value="AGPL-3.0-only"/>
```

Comment:
best to add a comment that they are only used for `minVersion>=31` else you risk breaking instances with older XSD versions